packages/flutter/lib/src/painting/shader_warm_up.dart

```diff
@@ -64,6 +66,8 @@ abstract class ShaderWarmUp {
   /// Trigger draw operations on a given canvas to warm up GPU shader
   /// compilation cache.
   ///
+  /// Parameter [image] is to be used for drawImage related operations.
```

Comment:
This is not grammatically correct.

But also, what parameter? The only parameter here is `canvas`.

Also, please hyperlink `drawImage` appropriately.